contrib/systemd/containerized-installs/openvswitch.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
Prefix it `openshift-openvswitch` and add a conflicts on `openvswitch` encase they're on a normal host where openvswitch rpm and service may already be installed?